webpack/assets/javascripts/react_app/components/Bookmarks/Bookmarks.js

```diff
@@ -6,96 +6,76 @@ import SearchModal from './components/SearchModal';
 import Bookmark from './components/Bookmark';
 import DocumentationUrl from '../common/DocumentationLink';
 import { STATUS } from '../../constants';
-import { bindMethods, noop } from '../../common/helpers';
+import { noop } from '../../common/helpers';
 import { sprintf, translate as __ } from '../../../react_app/common/I18n';
 
 class Bookmarks extends React.Component {
```

Comment:
should it be a simple component instead? e.g: `const Bookmarks = props => {...}`

Reply:
@laviro Updated